src/sim_engine/filesystem.rs

```diff
@@ -15,12 +17,12 @@ pub struct SimFilesystem {
     pub size: u64,
 }
 impl SimFilesystem {
-    pub fn new_filesystem(name: &str, mount_point: &str, size: u64) -> SimFilesystem {
+    pub fn new_filesystem(name: &str, mount_point: &str, size: Option<u64>) -> SimFilesystem {
         SimFilesystem {
             name: name.to_owned(),
             uuid: Uuid::new_v4(),
             mount_point: mount_point.to_owned(),
-            size: size,
+            size: size.unwrap_or(DEFAULT_FILESYSTEM_SIZE),
```

Comment:
Talked to @mulkieran in person - if this unwrap returns None, we want to set the default.  Looks good.